rpcd/playbooks/roles/rpc_maas/templates/client.my.cnf.j2

```diff
@@ -2,5 +2,5 @@
 
 [client]
 host={{ container_address }}
-user={{ galera_root_user }}
-password={{ galera_root_password }}
+user={{ galera_root_user|default('root') }}
+password={{ galera_root_password|default('secrete') }}
```

Comment:
I agree with cloudnull - I think its safe to assume people won't set the password, they should be forced to.